resources/faq.md

```diff
@@ -72,7 +72,7 @@ If you are curious about how the asynchronous part of MeiliSearch works, you can
 
 ## I am trying to add my documents but I keep receiving a `400 - Invalid data` response.
 
-MeiliSearch API only accepts JSON format.
+MeiliSearch API accepts JSON, CSV, and NDJSON formats.
```

Comment:
@gmourier, do we still get `400 — Invalid data` errors? According to the CSV spec, 400 should correspond to `Bad Request`. Or are these the same and largely dependent on stuff like browser implementation?

Reply:
This is legacy! Nice spot @guimachiavelli. I don't know when it changed probably before v0.20 🤓

The current error is 

```
{
    "message": "invalid type: map, expected a sequence at line 1 column 1. The json payload provided is malformed.",
    "errorCode": "bad_request",
    "errorType": "invalid_request_error",
    "errorLink": "https://docs.meilisearch.com/errors#bad_request"
}
```

and it will become a `malformed_payload` error code.

Reply:
Just to confirm we understand this: Is it possible a user gets a `400 — Bad Request` when adding documents due to a cause other than `malformed_payload`?

Reply:
Yes! `missing_payload` is also a `400 — Bad Request` HTTP code.

Reply:
updated